packages/chat-client/src/types/client.ts

```diff
@@ -9,23 +9,37 @@ export declare namespace ChatClientTypes {
   }
 
   // ---------- Data Types ----------------------------------------------- //
-  interface PartialInvite {
-    message: string;
-    account: string;
-    signature?: string;
+  interface Invite {
+    message: string; // character limit is 200. Must be checked by SDK before sending
```

Comment:
We might consider using [zod](https://www.npmjs.com/package/zod) for this project as TypeScript doesn't care about the string length while zod does and provides us type safety at runtime.

Reply:
That is actually very valid, might use this when we do the validation update. Thank you!